src/detail-panels/tab-selector.jsx

```diff
@@ -27,7 +27,15 @@ export function TabSelector({ className, tab, onTabSelected }) {
     'history':
       <VerticalTab key='history' className='tab-history'>
         {localise('History', { uk: 'Історія' })}
-      </VerticalTab>
+      </VerticalTab>,
+    'packs':
+    <VerticalTab key='packs' className='packs'>
+      {localise('Packs made', { uk: 'Історія' })}
```

Comment:
```suggestion
      {localise('Packs made', {})}
```
don't reuse a ukranian translation for a different phrase :stuck_out_tongue: